p2p/pex/pex_reactor_test.go

```diff
@@ -134,11 +134,11 @@ func TestPEXReactorReceive(t *testing.T) {
 
 	size := book.Size()
 	addrs := []*p2p.NetAddress{peer.NodeInfo().NetAddress()}
-	msg := cdc.MustMarshalBinary(&pexAddrsMessage{Addrs: addrs})
+	msg := cdc.MustMarshalBinaryBare(&pexAddrsMessage{Addrs: addrs})
```

Comment:
How are the two funcs differrent?

Reply:
Couldn't find it in the docs.

Reply:
MustMarshalBinary adds the uvarint length of the whole binary (before the actual data).